src/Tools/MicroBuild/Build.proj

```diff
@@ -10,17 +10,14 @@
 
   <!-- Non-official builds / local testing have different defaults -->
   <PropertyGroup Condition="'$(OfficialBuild)' != 'true'">
-    <UploadNugets>false</UploadNugets>
-    <UploadVsixes>false</UploadVsixes>
-    <UploadCoreXTPackages>false</UploadCoreXTPackages>
     <BranchName Condition="'$(BranchName)' == ''">master</BranchName>
     <SignRoslynArgs>-test</SignRoslynArgs>
     <PublishAssetsArgs>-test</PublishAssetsArgs>
     <CopyInsertionFileArgs>-test</CopyInsertionFileArgs>
   </PropertyGroup>
 
   <Target Name="Build">
-    <Exec Command="Restore.cmd /clean" WorkingDirectory="$(ProjectDir)" />
+    <Exec Command="Restore.cmd /clean" WorkingDirectory="$(ProjectDir)" Condition="'$(NoRestore)' == ''" />
```

Comment:
`Condition="'$(NoRestore)' != 'true'"`? Otherwise /p:NoRestore=false would do the wrong thing.